src/tests/ui/text_input.ts

```diff
@@ -8,11 +9,18 @@ export class TextInput extends BaseElement {
   }
 
   async enterText(text: string) {
-    await this.run('enter text', async()=> {
+    await this.run('enter text', async () => {
       await this.ready();
       await this.sendKeys(text);
-    })
+    });
     this.log('Entered Text');
   }
+
+  async pressEnter() {
+    await this.run('press enter', async () => {
+      await this.ready();
+      await this.sendKeys(protractor.Key.ENTER);
```

Comment:
import Key from protractor and then update here -
await this.sendKeys(Key.ENTER);